### PR TITLE
Update DepositData and Validator Proto Fields

### PR DESCRIPTION
**Pull request by terenc3t**
This is part of #2307 

Updated [validator](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/core/0_beacon-chain.md#validator) and [DepositData](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/core/0_beacon-chain.md#depositdata)  fields for Protobuf

### CODE REVIEW

**Comment by nisdas** [comment]
@terenc3t I changed the base branch to v0.6 from master

File: proto/beacon/p2p/v1/types.proto [hunk 1]
```diff
@@ -86,21 +86,32 @@ message AttestationDataAndCustodyBit {
...
+  uint64 activation_eligibility_epoch = 3;
+  // Epoch when validator activated
+  uint64 activation_epoch = 4;
+  // Epoch when validator exited
+  uint64 exit_epoch = 5;
+  // Was the validator slashed
+  bool slashed = 6;
+  // Rounded balance
+  uint64 high_balance = 7;
+  // Deprecated
```
Comment (prestonvanloon) [inline proto/beacon/p2p/v1/types.proto]:
```suggestion

  // Deprecated
  // TODO(2307): Remove deprecated fields
```

File: proto/beacon/p2p/v1/types.proto [hunk 2]
```diff
@@ -86,21 +86,32 @@ message AttestationDataAndCustodyBit {
...
 
 message Validator {
-  bytes pubkey = 1;
-  // TODO(781): The usage of withdrawal_credentials is not defined in spec. Not used in Prysm yet.
-  bytes withdrawal_credentials_hash32 = 2;
-  uint64 activation_epoch = 5;
-  uint64 exit_epoch = 6;
-  uint64 withdrawal_epoch = 7;
-  uint64 slashed_epoch = 8;
+  // BLS public key
```
Comment (prestonvanloon) [inline proto/beacon/p2p/v1/types.proto]: Many of these comments don't add additional information when the property name is descriptive enough.

Please use comments sparingly and consider removing redundant information.

File: proto/beacon/p2p/v1/types.proto (hunk 1 above)
Comment (prestonvanloon) [reply proto/beacon/p2p/v1/types.proto]: Can you also reorganize the deprecated fields so that they are in the same group?

**Pull request merged**